src/Page.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import Page from './App';
+
+test('Page', () => {
+    const tasks = [
+            {id: 1, title: 'Task-1'},
+            {id: 2, title: 'Task-2'},
+    ];
+
+    const {getByText} = render ((
+        <Page 
+            tasks={tasks}
+        />
+    ));
```

Comment:
```suggestion
    const {getByText} = render (<Page tasks={tasks} />);
```

줄바꿈을 하시는 기준이 있으신가요?

저는 `props` 가 2개 이상일 때만 줄바꿈을 해주는 편이에요. 

단순히 줄바꿈을 하더라도 나름의 기준이 있으면 좋을 것같아요.